frontend/src/main-bkp.scss

```diff
@@ -0,0 +1,346 @@
+@import "./reset.scss";
```

Comment:
Why do we have the -bkp files committed? we can always get the older version from Git, and I think we should just remove the bkp files.